crds/crd-coralogix-alert-set.yaml

```diff
@@ -78,6 +78,23 @@ spec:
                                 descprition: Sends resolve notification when the alert criteria that you defined is no longer met
                                 type: boolean
                                 default: true
+                              metricAlertPromqlParameters:
+                                description: Data for defining promql metric
+                                type: object
+                                properties:
+                                  promqlText:
+                                    type: string
+                                  arithmeticOperatorModifier:
+                                    type: integer
+                                    format: int32
+                                  sampleThresholdPercentage:
+                                    type: integer
+                                    format: int32
+                                  nonNullPercentage:
+                                    type: integer
+                                    format: int32
+                                  swapNullValues:
```

Comment:
I think we are missing some parameters that I can see in the UI like:
forOver,  ofTheLast, atLeast